samples/portability/cmsis_rtos_v2/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _cmsis_rtos_v2-sample:
+
+Dining Philosophers
```

Comment:
For the table of contents, since there's already a sample called "Dining Philosophers" change this doc's title to something like "Dining Philosophers (CMSI RTOS V2 APIs)"

Reply:
@dbkinder: Done, thanks.